full-stack-tests/backend/src/perftest/ElementCRUD.test.ts

```diff
@@ -82,11 +82,12 @@ function createElemProps(className: string, _iModelName: IModelDb, modId: Id64St
     elementProps.sub1Long = values.sub1Long;
     elementProps.sub1Double = values.sub1Double;
   }
-  if (className.includes("PerfElement2d"))
```

Comment:
This was a faulty behavior of our previous tests implementation. Fixed it, though it might slightly alter the tests output.